.golangci.yml

```diff
@@ -0,0 +1,123 @@
+# golangci.com configuration
+# https://github.com/golangci/golangci/wiki/Configuration
+service:
+  golangci-lint-version: 1.50.0 # use the fixed version to not introduce new linters unexpectedly
```

Comment:
This is pretty old, I think. We should be targeting something in the `1.57` range for Golang 1.22